### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by illright**
Finally everything seems fine with the CSS after the update. I'm planning to work on the statistics to see if any of it needs fixing, figured I'd do this first

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -48,14 +47,19 @@ export default {
       }),
       eslint(),
       svelte({
-        extensions: ['.html', '.svelte', '.svg'],
+        extensions: ['.html', '.svelte'],
         preprocess: preprocessChain,
         dev,
         hydratable: true,
       }),
+      // TODO: figure out the new image caching mechanism
```
Comment (aabounegm) [inline rollup.config.js:55]: What exactly about it is to be figured out?

**Review by aabounegm — COMMENTED** [review]
Did you test it or should I? Especially the Docker images

File: rollup.config.js (hunk 1 above)
Comment (illright) [reply rollup.config.js:55]: Well, do we implement it or not. I think we do, I'll add some more commits in a bit

Comment (illright) [reply rollup.config.js]: Nah, screw it

**Review by aabounegm — APPROVED** [review]

**Pull request merged**